### ISSUE TITLE
Default values during INSERT have different behavior in PROTOBUF vs other Formats

### ISSUE THREAD

**Issue by spena** [confluentinc/ksql#8996]
When I insert values on a Protobuf stream, the omitted fields, like Strings, have a default empty value.
```
ksql> create stream t1(id int, name string) with (kafka_topic='t1', value_format='protobuf', partitions=1);

 Message        
----------------
 Stream created 
----------------

ksql> insert into t1(id) values(1);

ksql> select * from t1;
+--------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------+
|ID                                                                                                |NAME                                                                                              |
+--------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------+
|1                                                                                                 |                                                                                                  |
Query Completed
```

But  when I do the same thing with Avro, JSON_SR, JSON and Delimited, the default value is null
```
ksql> create stream t2(id int, name string) with (kafka_topic='t2', value_format='avro', partitions=1);

 Message        
----------------
 Stream created 
----------------

ksql> insert into t2(id) values(1);

ksql> select * from t2;
+--------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------+
|ID                                                                                                |NAME                                                                                              |
+--------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------+
|1                                                                                                 |null                                                                                              |
Query Completed
```

Shouldn't PROTOBUF be consistent on all formats?

**Comment by suhas-satish** [comment]
@spena , worth trying to test with explicit null in a column if it works in proto. Based on that, we may need a doc update to explicitly state the behavior in ksqlDB.

**Comment by spena** [comment]
I did a test with `optional` fields and `default` fields in Protobuf and none of them allow nulls. The below protobuf schema returns `<empty>` if the `last` field does not have any value. Null is not allowed as default. And if I don't specify a default nor the last value, then the Protobuf API returns an empty string if the `last` field is missing. So no nulls are allowed in Protobuf.

```
syntax = "proto2";

message Source {
  required string name = 1;
  optional string last = 2 [default = '<empty>'];
}
```

I also tried to insert a `null` value to the stream, and the Protobuf result is empty.

There is an existing doc referencing how Protobuf cannot deal with null values. https://docs.ksqldb.io/en/latest/reference/serialization/#protobuf

I think the doc is enough and we cannot do much about fixing this inconsistency in KSQL. Users would need to understand the differences and write their use cases accordingly.
